testing/bdd/features/handover/handover_algorithm.feature

```diff
@@ -0,0 +1,50 @@
+@handover @node @active
+# When present in a document, the ebuttp:authorsGroupControlToken is a number that the
+# Handover Manager uses to identify which sequence to select: when a document is received with a
+# higher value ebuttp:authorsGroupControlToken than that most recently received in the
+# currently selected sequence the Handover Manager shall switch to that document's sequence, i.e. it
+# shall emit a document in its output sequence corresponding to the received document with the new
+# control token without delay.
+# Note: The above definition permits the control token value to be lowered after taking control,
+# either by setting the control token value to an explicit lower number or by omitting it in
+# subsequent documents and relying on the initial value of "0".
+
+Feature: Handover algorithm  
+
+  Examples:
+  | xml_file     | sequence_identifier | authors_group_identifier |
+  | handover.xml | handoverOutputSeq   | handoverTest01           |
+
+  
+  # SPEC-CONFROMANCE.md R23, R24, R25
+  # When a document is received with a higher value ebuttp:authorsGroupControlToken than that most recently received 
+  # in the currently selected sequence, the Handover Manager shall switch to that document's sequence without delay.
+  Scenario: Switch to higher value token and ignore lower value token
+    Given a handover node with <authors_group_identifier> and <sequence_identifier>
+    And an xml file <xml_file> 
+    When it has <sequence_identifier1> and <sequence_number1>
+    And it has <authors_group_identifier>
+    And it has <authors_group_control_token1>
+    And the document is generated
+    And handover node processes document
+    And new document is created
+    And it has <sequence_identifier2> and <sequence_number2>
+    And it has <authors_group_identifier>
+    And it has <authors_group_control_token2>
+    And the document is generated
+    And handover node processes document
```

Comment:
Well line 30 is `And new document is created` so that is the creation point, then down in line 34 we have `And the document is generated` - the difference between "creation" and "generation" is not obvious at all - why need both?

Then my issue with `And handover node processes document` is that in general in our code when a node is given a document to process, e.g. in a function like `processDocument()` that is on the input side, whereas for the handover manager it is actually on the output side. Is there a processing step for the handover manager after creating the document and before emitting it? I'm not sure what that would be - validation perhaps? Does it need to be in the BDD?